rules-survey/arrow-functions.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  rules: {
+    // https://eslint.org/docs/latest/rules/arrow-spacing
+    'arrow-spacing': 'error',
+
+    // https://eslint.org/docs/latest/rules/arrow-parens
+    'arrow-parens': 'error',
+
+    // https://eslint.org/docs/latest/rules/no-confusing-arrow
+    'no-confusing-arrow': ['error', {
```

Comment:
https://eslint.org/docs/latest/rules/no-confusing-arrow

Reply:
For a short condition, the arrow functions are just right. And for the long ones it's a nightmare